### PR TITLE
Apply TOC for every documentation

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/docs/issues/61

## TODOs
- [x] Loop each file of the documentation and apply the TOC

## Checklist (Developers)
- [x] Make this PR as a draft first
- [x] Title is checked
- [x] Background & TODOs are filled
- [x] Assignee is set
- [x] Labels are set
- [x] Project is created & linked, if multiple PRs are associated to this PR
- [x] Appropriate issue(s) is(are) linked to this PR under `development`
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
Please copy and paste the following into your review, ensuring each checkbox is marked as checked
```
- [ ] Check if there are any other missing TODOs that are not yet listed
- [ ] Review Code
- [ ] Every item on the checklist has been addressed accordingly
- [ ] If `development` is associated to this PR, you must check if every TODOs are handled
```

### CODE REVIEW

**Comment by akanex115** [comment]
更新機能により、不要な編集があるかもしれません。

File: install/eks_eksctl/04_ingress.md [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 <!-- /TOC -->
 
+<!-- /TOC -->
+
```
Comment (mlajkim) [inline install/eks_eksctl/04_ingress.md:17]:
```suggestion
```
Not necessary

File: install/eks_eksctl/05_create-secret.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 
 # Secret Yaml Setup
-
```
Comment (mlajkim) [inline install/eks_eksctl/05_create-secret.md:3]: dont delete line

File: install/eks_eksctl/07_kubectl-apply.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
   - [How to check](#how-to-check)
 
 <!-- /TOC -->
+<!-- /TOC -->
```
Comment (mlajkim) [inline install/eks_eksctl/07_kubectl-apply.md:11]: dont make unnecessary change

File: install/plantuml/install-plantuml.md [hunk 4]
```diff
@@ -16,7 +16,7 @@
     - [Done](#done)
   - [Run Remotely](#run-remotely)
     - [Set remote setting](#set-remote-setting)
-    - [Done](#done-1)
+    - [Done](#done)
```
Comment (mlajkim) [inline install/plantuml/install-plantuml.md:19]: roll back

File: install/local_mac_os/05_open_browser.md [hunk 5]
```diff
@@ -9,6 +9,8 @@
 
 <!-- /TOC -->
 
+<!-- /TOC -->
+
```
Comment (mlajkim) [inline install/local_mac_os/05_open_browser.md:13]: roll back

File: install/local_mac_os/03_fe.md [hunk 6]
```diff
@@ -10,6 +10,7 @@
     - [Run Frontend](#run-frontend)
 
 <!-- /TOC -->
+<!-- /TOC -->
```
Comment (mlajkim) [inline install/local_mac_os/03_fe.md:13]: roll back

**Review by mlajkim — COMMENTED** [review]
I think your tool is not working. give me a sec.

File: install/eks_eksctl/05_create-secret.md (hunk 2 above)
Comment (akanex115) [reply install/eks_eksctl/05_create-secret.md:3]: Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)

File: install/eks_eksctl/07_kubectl-apply.md (hunk 3 above)
Comment (akanex115) [reply install/eks_eksctl/07_kubectl-apply.md]: Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)

File: install/plantuml/install-plantuml.md (hunk 4 above)
Comment (akanex115) [reply install/plantuml/install-plantuml.md]: Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)

File: install/local_mac_os/05_open_browser.md (hunk 5 above)
Comment (akanex115) [reply install/local_mac_os/05_open_browser.md]: Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)

File: install/local_mac_os/03_fe.md (hunk 6 above)
Comment (akanex115) [reply install/local_mac_os/03_fe.md]: Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)

**Comment by akanex115** [comment]
I ran ⌘+S on all files, but none were updated.

**Review by mlajkim — COMMENTED** [review]
Please review your changes here
https://github.com/ajktown/docs/pull/65/files

and dont make unnecessary indents

**Review by mlajkim — APPROVED** [review]
OK
